src/test/transaction_tests.cpp

```diff
@@ -955,6 +955,15 @@ BOOST_AUTO_TEST_CASE(test_IsStandard)
     BOOST_CHECK(!IsStandardTx(CTransaction(t), reason));
     BOOST_CHECK_EQUAL(reason, "bare-multisig");
     fIsBareMultisigStd = DEFAULT_PERMIT_BAREMULTISIG;
+
+    // Check Taproot outputs dust threshold
+    t.vout[0].scriptPubKey = CScript() << OP_1 << ParseHex("72ad3fb702bf1a2111b09c2bf1e72f4f52d4ceb2acdcfcd0c63abf70a59c36d6");
```

Comment:
You can just use a repeating hex character to allow for better compression. 

```suggestion
    t.vout[0].scriptPubKey = CScript() << OP_1 << ParseHex("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
```

Also, would be nice to add a test for OP_16 (as you change this too in this pull) and OP_0, because it is missing.

Reply:
Used repeating characters, tested for Segwit v0, and added test for future witness versions.

Awkward to test something that is almost certainly wrong, comforts me in the idea that we should refactor `GetDustThreshold` to loudly fail rather than returning an invalid value.